src/osx/cocoa/textctrl.mm

```diff
@@ -1387,7 +1408,6 @@ - (void)controlTextDidEndEditing:(NSNotification *)aNotification
     [m_textField setDelegate: tf];
     m_selStart = m_selEnd = 0;
     m_hasEditor = [w isKindOfClass:[NSTextField class]];
-
```

Comment:
Sorry about this, but after submission I received an email that several tests failed because of wrong whitespaces, so I corrected them so the tests could pass. But I did not consciously change anything at the location posted above, maybe it was a merge issue?

Reply:
> Thanks!
> 
> Looks good to me, please let me know if you have any objections to the proposed changes, otherwise I can make them myself and merge. Or you could push another commit if you prefer to make them yourself, of course.
> 
> Thanks again!

Feel free to do the changes, they sound plausible!